packs/paizo-pregens/jaithe.json

```diff
@@ -3009,7 +3009,7 @@
                         "flag": "fleshwarpSize",
                         "key": "ChoiceSet",
                         "prompt": "PF2E.SpecificRule.Prompt.CreatureSize",
-                        "selection": "medicine"
+                        "selection": "med"
```

Comment:
Mind getting the AE-like as well?

Reply:
https://github.com/foundryvtt/pf2e/pull/15004/files

Reply:
Do you mean converting the AELike to CreatureSize as part of an unrelated migration? I'm not against it since migrations are a bit hacky regardless, but just making sure.

Reply:
No just in that file

Reply:
If its only to update the json manually with the updated value, done.